src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeListCasePersonalisation.java

```diff
@@ -91,7 +107,8 @@ public String getReferenceId(Long caseId) {
             .put("linkToOnlineService", iaExUiFrontendUrl)
             .put("hearingDate", dateTimeExtractor.extractHearingDate(hearingDateTime))
             .put("hearingTime", dateTimeExtractor.extractHearingTime(hearingDateTime))
-            .put("hearingCentreAddress", hearingCentreAddress);
+            .put("hearingCentreAddress", hearingCentreAddress)
+            .put("remoteVideoCallTribunalResponse", asylumCase.read(REMOTE_VIDEO_CALL_TRIBUNAL_RESPONSE, String.class).orElse(""));
```

Comment:
Same as above. We should put this in the `PersonalisationProvider` class as it is to do with hearing requirements. See the `buildHearingRequirementsFields(asylumCase, caseListingValues)` method.

Reply:
fixed